state/presence/presence.go

```diff
@@ -739,7 +843,7 @@ func (p *Pinger) ping() (err error) {
 		}
 		p.delta = delta
 	}
-	// TODO(perrito666) 2016-05-02 lp:1558657
+	// TODO(perrito667) 2016-05-02 lp:1558657
```

Comment:
I wonder if this was hitting the vim magic key for 'increment the number i'm on'. weird.